### ISSUE TITLE
[Bug]: flutter_blue_plus 1.12.13 code error published to pub.dev

### ISSUE THREAD

**Issue by techwnn** [boskokg/flutter_blue_plus#481]
### FlutterBluePlus Version

1.12.13

### Flutter Version

3.10.6

### What OS?

Android

### OS Version

Android 12

### What happened?

Android can't run example.

### Logs

```shell
145 lines in FlutterBluePlusPlugin error code like this:

mMtu.clear;
``` 
I think it should be:
```
mMtu.clear();
```
```

**Comment by techwnn** [comment]
@boskokg Please check the latest release, I think error code is published to pub.dev.

**Comment by chipweinberger** [comment]
pretty sure this was already fixed in 1.12.13
